docker-compose.yml

```
@@ -0,0 +1,46 @@
+version: "3.8"
+services:
+
+  scheduler:
+    image: ocurrent/ocluster-scheduler:live
+    user: "1001:1002"
+    command:
+      - --secrets-dir=/local.data/capnp-secrets
+      - --capnp-secret-key-file=/local.data/capnp-secrets/key.cap
+      - --capnp-listen-address=tcp:0.0.0.0:9000
+      - --capnp-public-address=tcp:scheduler:9000
+      - --pools=solver
+      - --state-dir=/local.data/ocluster-scheduler
+    init: true
+    ports:
+      - "9000:9000"
+    volumes:
+      - 'local.data:/local.data'
+
+  solver-worker:
+    build:
+      dockerfile: Dockerfile
+      context: .
+    user: "1001:1002"
+    command:
+      - --connect=/local.data/capnp-secrets/pool-solver.cap
+      - --name=solver-1
+      - --capacity=10
```

This capacity amount would relate to the resources allocated to docker. 
e.g.
```suggestion
      - --capacity=10 # The capacity of this solver worker, usually the number of concurrent processes it can run.
```

---

This is the number of builds that can run in parallel a typical worker. This is linked to the scheduler, different to the internal workers processes.